src/openrct2/actions/GameActionCompat.cpp

```diff
@@ -237,4 +238,35 @@ extern "C"
 
 #pragma endregion
 
+#pragma region PlacePeepSpawn
+    money32 place_peep_spawn(sint16 x, sint16 y, sint32 z, sint32 direction)
+    {
+        auto gameAction = PlacePeepSpawnAction(x, y, z, direction);
+        auto result = GameActions::Execute(&gameAction);
+        if (result->Error == GA_ERROR::OK)
```

Comment:
One of the plans with game actions is that we won't use money32 undefined as an error. Just return true or false.